src/components/SuppliersToBeCreated.jsx

```diff
@@ -0,0 +1,49 @@
+import { EntityStatusBadge, Table, TextLink } from "@contentful/f36-components";
+import { nanoid } from "nanoid";
+import { useSelector } from "react-redux";
+import { getSuppliersNotInContentful } from "../selectors";
+import React from "react";
+import LoadingTableCell from "./LoadingTableCell";
+import { useSDK } from "@contentful/react-apps-toolkit";
+import UpdateResult from "./UpdateResult";
+
+const SuppliersToBeCreated = () => {
+  const suppliersToBeCreated = useSelector(getSuppliersNotInContentful);
+  const sdk = useSDK();
+
+  if (suppliersToBeCreated.length === 0) {
+    return null;
+  }
+
+  return (
+    <React.Fragment>
+      {suppliersToBeCreated.map((pair) => (
+        <Table.Row key={nanoid()}>
```

Comment:
is the supplier name unique? This might make a better key? (I'm not sure if nanoid will give a new ID ever re-render which can lead to some issues like losing focus in accessibility etc if it thinks it's a new row)